File: Instruções Sequenciais/Q07.cpp
/*
------------------------------------------------
7. Crie um algoritmo/fluxograma/programa que 
declare uma variável do tipo ponto flutuante e 
atribua a ela um valor informado pelo usuário. 
Em seguida, arredonde esse valor para duas casas
decimais e imprima o resultado. Usar a 
biblioteca iomanip.
 */
#include <iostream>
#include <iomanip>
#include <locale.h>
using namespace std;
int main () {
	setlocale(LC_ALL,"Portuguese");
	float valor;
	cout<<"Informe um valor real:";
	cin>>valor;
	cout<<fixed<<setprecision(2)<<valor;
	return 0;
}

File: exercicios/python/Instruções Sequenciais/Q07.py
# Exercício 7 — Arredondando valor de ponto flutuante a 2 casas decimais
def main():
    valor = float(input("Informe um valor decimal: "))
    arredondado = round(valor, 2)
    print(f"Valor arredondado (2 casas decimais): {arredondado:.2f}")

if __name__ == "__main__":
    main()